DataManagementSystem/private/FTS3Utilities.py

```diff
@@ -322,22 +316,18 @@ def _randomServerPolicy( self, _attempt ):
 
     return fts3Server
 
-
-  @staticmethod
-  def _getFTSServerStatus( ftsServer ):
+  def _getFTSServerStatus(self, ftsServer):
     """ Fetch the status of the FTS server from RSS """
-    pub = RPCClient( 'ResourceStatus/Publisher' )
-    res = pub.getElementStatuses( 'Resource', ftsServer, None, None, None, None )
 
+    res = self._resourceStatus.getElementStatus(ftsServer, 'FTS')
     if not res['OK']:
       return res
 
-    if not res['Value']:
-      return S_ERROR( "No FTS Server %s known to RSS" % ftsServer )
-
-    rssDict = dict( zip( res['Columns'], res['Value'][0] ) )
+    result = res['Value']
+    if ftsServer not in result:
+      return S_ERROR("No FTS Server %s known to RSS" % ftsServer)
 
-    if rssDict['Status'] == 'Active':
+    if result[ftsServer]['all'] == 'Active':
```

Comment:
RSS returns lower case keys now ?!  @fstagni RSS returns capitalized keys for the SE. Should we try to have it consistent ?

Reply:
This `all` is for the status type (what would be, e.g., `ReadAccess` and `WriteAccess` in the SE status types). `all` is a special case.

Reply:
And does it need to be a special case for the capital as well :-(